web/analytics/export.py

```
@@ -0,0 +1,615 @@
+        writer.writerow([])
+        
+        # Write weekly activity distribution
+        writer.writerow(['Weekly Activity Distribution'])
+        writer.writerow(['Day', 'Activity Percentage'])
+        
+        for day, percentage in patterns_data['weekly_activity'].items():
+            writer.writerow([day, f"{percentage:.2f}%"])
+    
+    return response
+
+def export_pdf(request, analytics_type, obj_id=None):
```

Ruff (0.8.2)

`export_pdf` is too complex (12 > 10)

(C901)